l10n_br_delivery_nfe/tests/test_delivery_nfe.py

```diff
@@ -0,0 +1,207 @@
+# Copyright (C) 2024 Diego Paradeda - KMEE
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo.tests.common import SavepointCase
+
+
+class TestDeliveryNFe(SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.invoice_model = cls.env["account.move"]
+        cls.invoice_wizard = cls.env["stock.invoice.onshipping"]
+        cls.prod1 = cls.env.ref("product.product_product_12")
+        cls.prod2 = cls.env.ref("product.product_product_16")
+
+    def _change_user_company(self, company):
+        self.env.user.company_ids += company
+        self.env.user.company_id = company
+
+    # Testing Lucro Presumido - wo pack
+    def test_invoicing_picking_volume_lucro_presumido(self):
+        """Test Invoicing Picking NFe volume - Lucro Presumido"""
+        self._change_user_company(self.env.ref("l10n_br_base.empresa_lucro_presumido"))
+        picking = self.env.ref("l10n_br_stock_account.lucro_presumido-picking_1")
+
+        # Set product volume data
+        self.prod1.product_nfe40_esp = ""
+        self.prod1.product_nfe40_marca = "marca teste"
+        self.prod2.product_nfe40_esp = "esp2 teste"
+        self.prod2.product_nfe40_marca = "marca2 teste"
+        self.prod1.weight = 1
+        self.prod2.weight = 3
+        self.prod1.net_weight = 1
+        self.prod2.net_weight = 5
+
+        # Invoice
+        picking.set_to_be_invoiced()
+        picking.action_confirm()
```

Comment:
Recentemente foi incluído no l10n_br_stock_account uns Testes Comuns/Common Test para evitar código duplicado, diversos testes repetiam os mesmo processos https://github.com/OCA/l10n-brazil/blob/14.0/l10n_br_stock_account/tests/common.py com isso você pode remover o método _change_user_company e os processos de aprovar um Picking e criar a Fatura, um exemplo de como ser feito está aqui https://github.com/akretion/l10n-brazil/blob/14.0-REF-adapt_sale_stock_to_international_case/l10n_br_sale_stock/tests/test_sale_stock.py fazendo um from odoo.addons.l10n_br_stock_account.tests.common import TestBrPickingInvoicingCommon você pode usar os métodos 
        self.picking_move_state(picking)
        invoice = self.create_invoice_wizard(picking)
        Ou 
        pickings = picking | picking3 | picking4
        invoices = self.create_invoice_wizard(pickings)
        picking_devolution = self.return_picking_wizard(picking)
       self._change_user_company(self.env.ref("l10n_br_base.empresa_lucro_presumido"))
Em alguns testes acaba sendo necessário criar o wizard para validar campos, como esse teste especifico, mas quando for possível e melhor usar os métodos para evitar código duplicado.